stores/SettingStore.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import compareVersions from 'compare-versions';
+
+// TODO: Fix this import
+// Eslint was complaining about this import, maybe because it's coming from
+// an outdated module?  In any case, I think updating stuff is out of scope
+// for the PR in which this is being added.
+
+// eslint-disable-next-line import/namespace
+import { Platform } from 'react-native';
```

Comment:
eslint complains about a syntax error in this lib/file. I suspect it's related to this module being `ts` where the others where `Platform` is imported are `js`, and maybe an underlying library needs to be updated, idk.

This is the eslint error:

```
8:26  error    Parse errors in imported module 'react-native': ';' expected. (14:32)  import/namespace
```

Reply:
https://github.com/jellyfin/jellyfin-expo/issues/365

Reply:
Aye - I’ll peel out this comment then lol 😅

Reply:
Done. Eslint still ignoring so it gets through the build but I've tagged it with the issue number.